portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestACSide.testcase

```diff
@@ -201,4 +201,73 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-14070 | Automation ID: LRAC-14573 | Test Summary: Can publish variant from a terminated test and delete test in AC"
+    @priority = 5
+    test CanPublishVariantFromTerminatedTestAndDeleteTest {
+        task ("Edit the variant created") {
+            ABTest.editVariant(variantName = "Variant name");
+        }
+
+        task ("Add a heading to the page to diferentiate from the control") {
+            PageEditor.addFragment(
+                collectionName = "Basic Components",
+                fragmentName = "Heading");
+        }
+
+        task ("Click to save the variant") {
+            ABTest.saveVariant();
+        }
+
+        task ("Click to open the AB test panel") {
+            ABTest.openABTestPanel();
+
+            task ("Click to review and run the test") {
+                ABTest.runTest();
+            }
+        }
+
+        task ("Click to terminate the test") {
+            Button.click(button = "Terminate Test");
+
+            ABTest.confirmTerminateTest();
+        }
```

Comment:
Hey @manuferreira, I don't think it's necessary to check the modal text in these tests since it's not the purpose of the test. We have this case that already check this modal:

- CheckTerminateButtonRedirectingToDXP

Putting all the tests to make this view may not be very good, since if we have a change in the text, then all the tests will fail. As we have this test for the purpose of checking the redirect, I think it is the best case to validate this message whenever it is executed. What do you think?

Reply:
I think you could use:

```
task ("Terminate AB Test") {
	ACTests.terminateABTest(testName = ${nameABTest});
}
```

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/a4b20ba1958b33ce52abb05f6e1af7a78865b1f0